CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All released to this project will be documented in this file.
```

Comment:
Change to "All notable changes to this project will be documented in this file."